tests/functional/adapter/test_simple_seed.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+from dbt.tests.adapter.simple_seed.fixtures import macros__schema_test
+from dbt.tests.adapter.simple_seed.seeds import (
+    seeds__enabled_in_config_csv,
+    seeds__tricky_csv
+)
+from dbt.tests.adapter.utils.base_utils import run_dbt
+
+
+_SEED_CONFIGS_CSV = """
+seed_id,stuff
+1,a
+2,b
+""".lstrip()
+
+_SCHEMA_YML = """
+version: 2
+seeds:
+- name: seed_enabled
+  columns:
+  - name: birthday
+    tests:
+    - column_type:
+        type: STRING
+  - name: seed_id
+    tests:
+    - column_type:
+        type: FLOAT64
+
+- name: seed_tricky
+  columns:
+  - name: seed_id
+    tests:
+    - column_type:
+        type: INT64
+  - name: seed_id_str
+    tests:
+    - column_type:
+        type: STRING
+  - name: a_bool
+    tests:
+    - column_type:
+        type: BOOLEAN
+  - name: looks_like_a_bool
+    tests:
+    - column_type:
+        type: STRING
+  - name: a_date
+    tests:
+    - column_type:
+        type: DATETIME
+  - name: looks_like_a_date
+    tests:
+    - column_type:
+        type: STRING
+  - name: relative
+    tests:
+    - column_type:
+        type: STRING
+  - name: weekday
+    tests:
+    - column_type:
+        type: STRING
+
+- name: seed_configs
+  config:
+    hours_to_expiration: 2
+    labels:
+      contains_pii: 'yes'
+      contains_pie: 'no'
+""".lstrip()
+
+
+class TestSimpleSeedConfigs:
```

Comment:
It is named configs versus the base class `BaseSimpleSeedColumnOverride` so it seems a bit strange to inherit.

Reply:
can we inherit from the `SeedConfigBase` class?